ironweb.d.ts

```diff
@@ -15,6 +15,16 @@ export interface DocumentCreateOptions {
     documentName?: string;
     accessList?: DocumentAccessList;
     grantToAuthor?: boolean;
+    policy?: Policy;
+}
+/**
+ * A policy which can be used to determine the users and groups to share with.
+ */
+export interface Policy {
+    classification?: string;
+    category?: string;
+    dataSubject?: string;
+    substituteId?: string;
```

Comment:
In IronOxide these names are different. They should probably be the same.

Reply:
ok.